deployment/docker-compose.yml

```diff
@@ -34,7 +34,7 @@ services:
     env_file:
       - .env
     volumes:
-      - "./mqtt2influx-configuration.yaml:/mqtt2influx-configuration.yaml"
+      - "./mqtt2influx-configuration-prod.yaml:/mqtt2influx-configuration.yaml"
```

Comment:
The name of the local file is the users responsibility.  I would leave it the same